dimension/areas.go

```diff
@@ -0,0 +1,13 @@
+package dimension
+
+// Area is an area model with ID and Label
+type Area struct {
+	ID       string `json:"id"`
+	Label    string `json:"label"`
+	AreaType string `json:"area-type"`
```

Comment:
I think we use snake_case for json fields

Reply:
Here it seems to be kebab-case
https://github.com/ONSdigital/dp-cantabular-dimension-api/blob/cf38fd889eeffd2243149995ac0f61b9eeb5480d/contract/areas.go#L8